evm/contracts/bridge/TellorDataBridge.sol

```diff
@@ -47,6 +46,9 @@ contract TellorDataBridge is ECDSA {
     address public deployer; /// Address that deployed the contract.
     bool public initialized; /// True if the contract is initialized.
     uint256 public constant MS_PER_SECOND = 1000; // factor to convert milliseconds to seconds
+    bytes32 public constant NEW_REPORT_ATTESTATION_DOMAIN_SEPARATOR = // "tellorCurrentAttestation"
```

Comment:
want to put a note in as far why it's changing and what it is for the different testnets?